ucgenerator/models.go

```diff
@@ -197,7 +197,7 @@ func (o *object) isArray() bool {
 }
 
 func (o *object) isScalar() bool {
-	return !(o.isObject() || o.isArray())
+	return !o.isObject() && !o.isArray()
```

Comment:
So this is applies to De Morgan's law (and all other cases) https://en.wikipedia.org/wiki/De_Morgan%27s_laws
Is this changed happed by any linter? I mean, do we need to write conditions always this way?

Reply:
Yes, it is. This is a part of [staticcheck](https://golangci-lint.run/usage/linters/#staticcheck) linter which we had before. 
For my taste it makes sense, if you don't like it in general, we can disable this particular linter, or you can add `//nolint:staticcheck` to the place where it doesn't work on your opinion.